src/material/chips/chip.ts

```diff
@@ -243,8 +252,15 @@ export class MatChip extends _MatChipMixinBase implements FocusableOption, OnDes
 
     this._addHostClassName();
 
-    this._chipRipple = new RippleRenderer(this, _ngZone, _elementRef, platform);
+    // Dynamically create the ripple target, append it within the chip, and use it as the
+    // chip's ripple target. Adding the class '.mat-chip-ripple' ensures that it will have
+    // the proper styles.
+    this._chipRippleTarget = this._document.createElement('div');
+    this._chipRippleTarget.classList.add('mat-chip-ripple');
+    this._elementRef.nativeElement.appendChild(this._chipRippleTarget);
+    this._chipRipple = new RippleRenderer(this, _ngZone, this._chipRippleTarget, platform);
```

Comment:
We should call `_removeTriggerEvents` on the `_chipRipple` on destroy.

Reply:
It's already called? See `MatChip.ngOnDestroy`.

Reply:
Ah, I assumed that the `_chipRipple` was added with these changes and I didn't see it in the diff.